awx/main/migrations/0175_db_monitoring.py

```diff
@@ -0,0 +1,175 @@
+# Generated by Django 3.2.16 on 2023-02-07 14:39
+
+import os
+
+from django.db import connection, migrations
+
+
+class ConfigurationError(Exception):
+    pass
+
+
+def db_monitor_role_exists(conn):
+    sql = """
+select (to_regrole(%s) is not null)::bool as "role_exists" ;
+    """
+    cur = conn.cursor()
+    cur.execute(sql, ('db_monitor',))
+    res = cur.fetchone()
+    cur.close()
+    return res[0]
+
+
+def validate_pg_stat_statements_preloaded(conn):
+    sql = """
+select setting
+  from pg_settings
+ where name = %s
+ ;
+    """
+    cur = conn.cursor()
+    cur.execute(sql, ('shared_preload_libraries',))
+    res = cur.fetchone()
+    cur.close()
+    if 'pg_stat_statements' not in res[0]:
+        raise ConfigurationError("The 'shared_preload_libraries' setting must include 'pg_stat_statements'")
```

Comment:
This requires a db configuration change, right? Making this a fatal error seems like it would inflict much pain and suffering upon both us and our support team. Let me know if I'm missing something here.

Reply:
This migration has been removed.